api/src/main/kotlin/com/openlattice/collaborations/ProjectedTableMetadata.kt

```diff
@@ -0,0 +1,8 @@
+package com.openlattice.collaborations
+
+import java.util.*
+
+data class ProjectedTableMetadata(
```

Comment:
Another table metadata class?

Reply:
happy to rename to something else, but yeah we need a way to track which tables are projected to which collaborations.